### DISCUSSION TITLE
Budget and hidden Sub categories

### DISCUSSION THREAD

**Discussion by ghost** [moneymanagerex/moneymanagerex#4835] [Ideas]
Could we have an option to hide the "hidden" subcategories in budget ?

From my side, i'd love to have  an option to hide unreconciled transcations from budget also ! :)

![image](https://user-images.githubusercontent.com/81561095/179522754-a81748dd-5ee3-4679-9c01-ce31ab8a31a8.png)

categories "(archive)" are supposed to be hidden

**Comment by vomikan** [comment]
To do it well, you need to do it after completing the case #4216
